substrate/client/rpc-servers/src/lib.rs

```diff
@@ -98,9 +106,9 @@ struct PerConnection<RpcMiddleware, HttpMiddleware> {
 }
 
 /// Start RPC server listening on given address.
-pub async fn start_server<M>(
+pub async fn start_server<M: Send + Sync + 'static>(
```

Comment:
put it in the where clause or remove the where clause below.

Reply:
Good catch. I had manually cherry-picked this code from our fork of the old substrate repo. Fixed it.